services/venice-controller/src/main/java/com/linkedin/venice/controller/server/StoragePersonaRoutes.java

```diff
@@ -77,7 +78,8 @@ public void internalHandle(Request request, StoragePersonaResponse veniceRespons
         } catch (Exception e) {
           veniceResponse.setError(
               "Failed when getting persona " + personaName + ". Exception type: " + e.getClass().toString()
-                  + ". Detailed message = " + e.getMessage());
+                  + ". Detailed message = " + e.getMessage(),
```

Comment:
Hmm, I see there is a lot of this kind of error message pattern "customized error msg + Exception type = xxx, Detailed message = xxx", wondering if we could do it in a single method and use it everywhere

Reply:
Yeah, we can